build/build.cake

```diff
@@ -258,30 +258,6 @@ Task("Test")
     };
 
     VSTest(baseDir + $"/**/{configuration}/**/UnitTests.*.appxrecipe", testSettings);
-}).DoesForEach(GetFiles(baseDir + "/**/UnitTests.*NetCore.csproj"), (file) =>
-{
-    Information("\nRunning NetCore Unit Tests");
```

Comment:
This doesn't seem correct, wouldn't this also cause `UnitTests.Notifications.NetCore` to be skipped?

Reply:
UnitTests .NET Core cake section brought back